recipes/agent_windows.rb

```diff
@@ -1,3 +1,32 @@
 # Installs and configures the New Relic Infrastructure agent on Windows
+#
+# Chef uses sha256 checksums, to generate this, I downloaded the file
+# and on OS X did 'shasum -a 256 newrelic-infra.msi' YMMV on other
+# platforms
+#
+# TODO: pin to specific version
+# TODO: abstract checksum
+# TODO: make install location configurable
 
-raise 'Windows chef support for New Relic Infrastructure agent not yet available'
+windows_package 'newrelic-infra' do
+  action :install
+  source 'https://download.newrelic.com/infrastructure_agent/windows/newrelic-infra.msi'
+  checksum '3c9f98325dc484ee8735f01b913803eaef54f06641348b3dd9f3c0b3cd803ace'
```

Comment:
Thanks @donone-apex. I've pinged some other folks internally on this one. Trying to figure best way to generalize this without immediately requiring folks to have to write similar wrapper.  

Could change the checksum to be blank (or a nil attribute) to start, then your local code could fill that in. For people using the generic version of the recipe, it would out of the box work for the latest version.

I'm not super familiar with the `windows_package` best practice, so are there alternatives?

This is really the only blocker to merging this.